tools/fastq_info/macros.xml

```diff
@@ -0,0 +1,31 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.25.1</token>
+    <token name="@SUFFIX_VERSION@">0</token>
+    <xml name="edam_ontology">
+        <edam_topics>
+            <edam_topic>topic_0091</edam_topic>
+        </edam_topics>
+        <edam_operations>
+            <edam_operation>operation_0336</edam_operation>
+        </edam_operations>
+    </xml>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package">fastq_utils:@TOOL_VERSION</requirement>
```

Comment:
? what is that? theversion is specified in a different way ...

Reply:
True, sorry I used the bionano macros file as template, but it used Docker.